ArduPlane/RC_Channel.cpp

```diff
@@ -1,6 +1,7 @@
 #include "Plane.h"
 
 #include "RC_Channel.h"
+#include "qautotune.h"
```

Comment:
qautotune ?

Reply:
needed for QAUTOTUNE_ENABLED, so it can be enabled on custom build server